modules/core/src/lib/layer.js

```diff
@@ -193,6 +193,10 @@ export default class Layer extends Component {
     return false;
   }
 
+  needs64bitPositions() {
```

Comment:
Why do we need a wrapper for `is64bitEnabled` function? Can we consolidate?

Isn't the key change we need to remove the `if (this.props.fp64)` check from `is64bitEnabled`?